documentation/modules/auxiliary/scanner/discovery/udp_probe.md

```diff
@@ -0,0 +1,70 @@
+## Description
+
+The “udp_probe” module scans a given range of hosts for common UDP services. Note: This module is deprecated and may disappear at any time.
```

Comment:
It would be nice to know what ports it probes by default and have that documented under the description